rotatingtext/src/main/java/com/sdsmdg/harjot/rotatingtext/RotatingTextSwitcher.java

```diff
@@ -302,7 +321,7 @@ public void accept(Long aLong) throws Exception {
 
     private void updatePaint() {
         paint.setTextSize(rotatable.getSize() * density);
-        paint.setColor(rotatable.getColor());
+        //paint.setColor(rotatable.getColor());
```

Comment:
I see that you have commented out the `setColor` method call in line **73** and **324**. So now, there is just one place where all the functioning regarding changing of colors is taking place i.e. **Lines 155- 174** ,and that piece of code necessarily needs the use of a list of colors. This is not the right way and can cause problems for users while using the library. 

Your code must be structured in such a way that the **colorList should be used if and only if the user explicitly passes it**, otherwise the code should run as it was running before. Therefore you must not comment out these lines or if you absolutely need to do so then provide an alternate way to setColor without the use of a `colorList`.

@dev-ritik I would really appreciate your thoughts on this.

Reply:
There was no need to comment it out...I have adjusted the code .

Reply:
Okay